src/components/ClassCard.css

```diff
@@ -0,0 +1,18 @@
+.ClassCard {
+  width: 22rem;
```

Comment:
is that good practise to use rem here? i thought about using vw here, since it is more responsive?!

Reply:
I would say it is. It changes according to its parent font size. But maybe vw is also good. I would have to check. ;)